netdeploy/remote/deployedNetwork.go

```diff
@@ -404,6 +405,14 @@ func (cfg DeployedNetwork) GenerateCloudTemplate(templates HostTemplates, target
 		if err != nil {
 			return
 		}
+
+		group := strings.TrimSpace(cloudHost.Group)
+		if group == "" {
+			hostSpec.Group = "default"
```

Comment:
For backward compatibility, you might want to leave the default as an empty string.
This would force the cloudspec file consumer to populate it with "default" instead of expecting a non-empty string.
( i.e. currently, all cloud-spec files doesn't have this field. )

If you're going to break backward compatibility, please add versioning to the cloud spec file so we can detect that forward.

Reply:
Leave default as an empty string.